src/components/errorBoundary/ErrorBoundary.js

```diff
@@ -0,0 +1,11 @@
+import "./ErrorBoundary.css"
+
+const ErrorBoundary = (props) => {
```

Comment:
1. Why ErrorBoundary's folder in camelCase and not in PascalCase like the others folders? 
2. Why ErrorBoundary is functional component? (https://reactjs.org/docs/error-boundaries.html)